teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote><a href="https://pubchem.ncbi.nlm.nih.gov/" target="_blank">PubChem</a> is an open database of chemical molecules and their activities against biological assays, which <strong>is </strong>maintained by <a href="https://www.ncbi.nlm.nih.gov/" target="_blank">National Center for Biotechnology Information (NCBI)</a>, part of <a href="https://www.nih.gov/" target="_blank">National Institutes of Health (NIH)</a>. </blockquote><blockquote>There are three dynamically growing primary subdatabases, <strong>i.e. the </strong>compounds, substances, and bioassay databases</blockquote><p> </p><blockquote>With information about millions of compounds and their descriptions, everyone is welcome to use them.</blockquote><p>Information about millions of compounds is freely available.</p><p>(how is "description" different from "information"?)</p><p> </p><blockquote><s>In there, </s><strong>These </strong>compounds can be <s>searched by</s> <strong>queried using </strong>a broad range of properties</blockquote><p> </p><p>What do you mean by</p><blockquote>name fragments</blockquote><p>Do you mean "name, fragments" or "fragment names"?</p><p> </p><blockquote>It's no doubt</blockquote><p>"There is no doubt" maybe?</p><p> </p><blockquote>P<u>or</u>grammatic access to PubChem </blockquote><blockquote>several ways <s>for</s> <strong>of</strong> programmatic access</blockquote><p> </p><blockquote><a href="https://pubchemdocs.ncbi.nlm.nih.gov/pug-rest" target="_blank">PUG-REST</a>&nbsp;is a Representational State Transfer <strong>(REST)-style [Note: remove spaces]</strong> version of <strong>the </strong>PUG (Power User Gateway) web service, with both the syntax of the HTTP requests and the available functions. It can also provide convenient access to information on PubChem records which <strong>are </strong>not <s>possible</s> <strong>reachable </strong>with other PUG services.</blockquote><ul><li>"with both the syntax of the HTTP requests and the available functions" - available functions from where? REST or PUG?</li></ul><p> </p><blockquote>PUG <s>is</s> <strong>offers </strong>programmatic access to PubChem service via a single common gateway interface (CGI).</blockquote><p> </p><blockquote>RDF-encoded</blockquote><p>What is RDF?</p><p> </p><blockquote>In this tutorial, we will focus on the REST variant API, referring to talktorial<s>s</s> <a href="https://github.com/volkamerlab/teachopencadd/tree/master/teachopencadd/talktorials/T011_query_online_api_webservices" target="_blank">T011</a>. </blockquote><p>In this tutorial, we will focus on the REST variant API, <strong>please refer to talktorial</strong> <a href="https://github.com/volkamerlab/teachopencadd/tree/master/teachopencadd/talktorials/T011_query_online_api_webservices" target="_blank">T011</a> <strong>for an introduction</strong>. </p><p> </p><blockquote>For more details about <strong>the </strong>PubChem API, please <s>see</s> <strong>check out </strong>the introduction about <a href="https://pubchemdocs.ncbi.nlm.nih.gov/programmatic-access" target="_blank">programmatic access</a> <s>from</s> <strong>in the</strong> PubChem Docs.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I will fix the grammar problem</p>